client/build

```diff
@@ -1,12 +1,17 @@
 #!/bin/sh
 
+root_dir=$(git rev-parse --show-toplevel 2>/dev/null)
+gen_dir="$root_dir/client/gen"
+pushd "$root_dir/client"
+
 printf "\x1b[1;97mStart application build\n======================="
 printf "\x1b[m\n"
 echo
 
 suc=1
 
-mkdir gen &> /dev/null
+# TODO check if there's a better structure
+mkdir "$gen_dir" &> /dev/null
```

Comment:
I'd like to check if there's a better approach than having another artifact directory.